scripts/zones/Windurst_Waters/npcs/Leepe-Hoppe.lua

```diff
@@ -27,6 +27,7 @@ entity.onTrigger = function(player, npc)
     local MissionStatus = player:getCharVar("MissionStatus")
     local tuningIn = player:getQuestStatus(tpz.quest.log_id.WINDURST, tpz.quest.id.windurst.TUNING_IN)
     local tuningOut = player:getQuestStatus(tpz.quest.log_id.WINDURST, tpz.quest.id.windurst.TUNING_OUT)
+    local turmoil = player:getQuestStatus(tpz.quest.log_id.WINDURST, tpz.quest.id.windurst.TORAIMARAI_TURMOIL)
```

Comment:
file is missing
```
require("scripts/globals/quests")
require("scripts/globals/titles")
```

not your fault; these should have been required already, before your edits.  notice tpz.title is used on line 179 and elsewhere